wal/wal.go

```diff
@@ -417,13 +417,13 @@ func (w *WAL) cut() error {
 	fpath := filepath.Join(w.dir, walName(w.seq()+1, w.enti+1))
 
 	// create a temp wal file with name sequence + 1, or truncate the existing one
-	newTail, err := w.fp.Open()
+	tempTail, err := w.fp.Open()
```

Comment:
not sure if this variable name change accurately reflects what's happening; from the file system's perspective newTail == tempTail since the rename preserves the inode number